.github/workflows/eamxx-sa-testing.yml

```diff
@@ -48,15 +37,57 @@ env:
   submit: ${{ github.event_name == 'schedule' && 'true' || 'false' }}  # Submit to cdash only for nightlies
 
 jobs:
+  should_run_sa:
+    runs-on: ubuntu-latest  # This job runs can run anywhere
+    outputs:
+      value: ${{ steps.check.outputs.value }}
+    steps:
+      - name: Check paths
```

Comment:
I saw that, and thought about using it. But I don't know how it does that. I think that action has one of two choices: use gh API via curl, or clone repo and do git diff. Since I don't want to clone (takes too long), I may as well do the curl call myself.